ctapipe_io_magic/__init__.py

```diff
@@ -205,6 +275,8 @@ def __init__(self, input_url=None, config=None, parent=None, **kwargs):
         self.mars_datalevel = run_info[3][0]
 
         self.metadata = self.parse_metadata_info()
+        
+        self.laser = self.parse_laser_info()
```

Comment:
The parse of the laser info should be done only for real data (MC do not have that info), please add an if statement before the parsing.